pkgs/development/python-modules/yaswfp/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "yaswfp";
+  version = "0.9.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "138w6pnn75vd3ivykc7wsafhg1izjvq9bzz1398vnss7cxrhawb1";
+  };
+
+  # Tests are only available in repo
```

Comment:
Can we fetch that repo?

Reply:
We can and running the tests is possible.